packages/esm-patient-banner-app/src/require-context-polyfill.ts

```diff
@@ -0,0 +1,49 @@
+/**
+ * require.context is webpack magic or sugar to import individual modules in a given directory
+ * There are challenges when the code runs outside of webpack - such as jest or Node.
+ * One way to solve it is using bable transform-require-context plugin - but that requires depdnency on Babel
+ * Below code is just a pollyfill to do the same thing that react.context does - when react.context is unavailable
+ * Another approach could be to not use react.context and add the imports for traslations individually
+ *
+ * A good place for this pollyfil would be inside esm-utils
+ */
+
+export const requireContext = (base = '../translations', scanSubDirectories = false, regularExpression = /.json$/) => {
```

Comment:
If we're going to use a polyfill here, it really should be written:

```ts
export const requireContext = request.context !== 'undefined' ? require.context : (base, scanSubdirectories, regularExpression) =>
```

Reply:
> If we're going to use a polyfill here, it really should be written:
> 
> ```ts
> export const requireContext = request.context !== 'undefined' ? require.context : (base, scanSubdirectories, regularExpression) =>
> ```

yes that is the point, we should externalize require.context and not let it penetrate in all of the MF's index.js... I agree a polyfill is overly-done since outside of web-pack we wont need the functionality anyways...